src/controllers/webhook.controller.ts

```diff
@@ -21,17 +21,6 @@ export class WebhookController {
       );
 
       switch (type) {
-        case "NCW_DEVICE_MESSAGE": {
-          const { walletId, deviceId, physicalDeviceId, data } = req.body;
-          await handleNcwDeviceMessage(
```

Comment:
remove db table and routes etc

Reply:
reverted, i prefer creating a separate mr for removing all of these